heft-plugins/heft-dev-cert-plugin/src/webpack-dev-middleware-workaround.d.ts

```diff
@@ -4,6 +4,5 @@
 // Node 12 is still LTS so we need to support it.
 // Upstream issue: https://github.com/webpack/webpack-dev-middleware/issues/1194
 declare module 'fs' {
-  // eslint-disable-next-line
-  export type StatSyncFn = any;
+  export type StatSyncFn = typeof statSync;
```

Comment:
Is this workaround still necessary? I'm pretty sure the fix was published.

Reply:
Might not be, but there's no downside to having it.